src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -189,6 +189,39 @@
     type: command
     short-summary: List policy definitions.
 """
+helps['policy setdefinition create'] = """
```

Comment:
IMO `setdefinition` should be `set-definition` for all these new commands.

Reply:
Or even just `set` unless there are "set" commands and "set-definition" commands...

Reply:
set-definition sounds more apt. thanks